ChangeLog.d/disable-new-session-tickets.txt

```diff
@@ -0,0 +1,14 @@
+Bugfix
+   * Fix TLS connection failure in applications using an Mbed TLS client in
+     the default configuration connecting to a TLS 1.3 server sending tickets.
+     See the documentation of
+     mbedtls_ssl_conf_tls13_enable_signal_new_session_tickets() for more
+     information.
+     Fixes #8749.
+
+Changes
```

Comment:
This entry should be under “Default behavior changes”.

I only noticed just now and I don't want to go through another round of rework here, so let's fix it during release prep at the changelog polishing step.